base

```diff
@@ -55,7 +55,6 @@ initialize () {
 
 	rsync -a src/ cache
 
-	docker $DOCKER_OPTS pull $DOCKER_REGISTRY/afcowie/debian:jessie
```

Comment:
Why is this being removed? The jessie repo is used as the base of the final image